tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+import copy
+import json
+
+import ocdskit.oc4ids_transforms as transforms
+from tests import read
+
+
+def test_initial_tranform_state():
+    releases = json.loads(read("release-package_additional-contact-points.json"))["releases"]
+    transform_state = transforms.InitialTransformState(releases, "1")
+    assert len(transform_state.compiled_releases) == 1
+    assert len(transform_state.releases_by_ocid["ocds-213czf-1"]) == 2
+
+
+def test_run_all():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "roles": ["publicAuthority"]}],
+        }
+    ]
+    output = transforms.run_transforms({}, releases, "1")
+    assert output["parties"] == releases[0]["parties"]
+
+
+def test_public_authority_role():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "roles": ["publicAuthority"]}, {"id": "2", "roles": ["publicAuthority"]}],
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.public_authority_role])
+    assert output["parties"] == releases[0]["parties"]
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "roles": ["publicAuthority"]}, {"id": "2", "roles": ["buyer"]}],
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.public_authority_role])
+    assert len(output["parties"]) == 1
+
+
+def test_buyer_role():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "roles": ["buyer"]}],
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.buyer_role])
+
+    assert "publicAuthority" in output["parties"][0]["roles"]
+    assert "buyer" in output["parties"][0]["roles"]
+
+
+def test_sector():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {"project": {"sector": "a"}},
+        }
+    ]
+
+    output = transforms._run_transforms(releases, "1", transforms=[transforms.sector])
+    assert output["sector"] == "a"
+
+
+def test_additional_classifications():
```

Comment:
Should this test also cover the scenario where there are multiple contracting processes, each with a different list of `additionalClassifications`?